### PR TITLE
adds default command set for bot

### PR DESCRIPTION
**Pull request by rlogiacco**
Here is the idea: this function can be executed after connecting to telegram with no arguments (thus `force` is false) and the default commands are set only if no commands have ever been set on the bot.
An additional system or settings sub-command might be added which forces the setting of the bot commands to its defaults.

### CODE REVIEW

**Comment by giloser** [comment]
Hi!

seems good for me!
I would have maybe add a log in the catch exception just to be sure.

File: octoprint_telegram/__init__.py [hunk 1]
```diff
@@ -1262,6 +1262,42 @@ def test_token(self, token=None):
...
+				commands.append({"command":"user","description":"Gets user info"})
+				commands.append({"command":"gif","description":"Sends a gif from the current video"})
+				commands.append({"command":"supergif","description":"Sends a bigger gif from the current video"})
+				commands.append({"command":"shutup","description":"Disables automatic notifications until the next print ends"})
+				commands.append({"command":"dontshutup","description":"Makes the bot talk again (opposite of `/shutup`)"})
+				commands.append({"command":"help","description":"Shows this help message"})
+				resp = requests.post(self.bot_url + "/setMyCommands", data={'commands':json.dumps(commands)}).json()
+				self._logger.debug("setMyCommands returned " + str(resp))
+		except Exception as ex:
+			pass
```
Comment (rlogiacco) [inline octoprint_telegram/__init__.py:1299]: @giloser if you mean you would have added a log statement here, looks like in every other command handling function there is no such log and my guess is this exception occurs only if the bot is not connected (aka unable to send messages)....
@fabianonline do you want to add anything on this?

Comment (giloser) [reply octoprint_telegram/__init__.py:1299]: Ok for me then 👯

**Comment by giloser** [comment]
this has been add in #342 thanks again @rlogiacco

**Pull request closed without merging**